package/CHANGELOG

```diff
@@ -131,9 +131,13 @@ Enhancements
     'protein' selection (#2751 PR #2755)
   * Added an RDKit converter that works for any input with all hydrogens
     explicit in the topology (Issue #2468, PR #2775)
-  
+  * Added a ValueError raised when not given a gridcenter while providing
```

Comment:
*Nitpick:* Can you mention the unit in which you made the changes (DensityAnalysis)?